package.json

```diff
@@ -184,6 +184,7 @@
     "rimraf": "5.0.1",
     "simple-git": "3.19.1",
     "tiny-glob": "0.2.9",
+    "ts-node": "10.9.1",
```

Comment:
This dependency was missing but we actually have 3 different npm scripts depending on it which were failing, fixed now.